feature/aks-cluster-creation/vars.tf

```diff
@@ -0,0 +1,116 @@
+    description = "Public ssh key path"
+    default     = "~/.ssh/id_rsa.pub"
+}
+
+variable "nodes_public_ip" {
+    description = "Assign a public IP per node for your node pools. In order to allow that, You have to install latest aks-preview extension and register feature for Node Public IP following the url: https://docs.microsoft.com/en-us/azure/aks/use-multiple-node-pools#assign-a-public-ip-per-node-for-your-node-pools-preview "
+    default     = false
+}
+
+variable "network_plugin" {
+    description = "azure or kubenet supported only, if network policy set to azure, network plugin must be set to azure"
+    default     = "azure"
+}
+
+variable "network_policy" {
+    description = "azure or calico supported, when Azure is set, network plugin must be set Azure value too"
+    default     = "azure"
+}
+
+variable "linux_admin_username" {
+    description = "Admin user on Linux OS"
+    default     = "operation"
+}
+
+variable "default_node_pool" {
+  description = "The object to configure the default node pool with number of worker nodes, worker node VM size and Availability Zones."
+  type = object({
+    name                           = string
+    node_count                     = number
+    vm_size                        = string
+    os_disk_size_gb                = string
+    type                           = string
+    enable_auto_scaling            = bool
+    min_count                      = number
+    max_count                      = number
+  })
+    default = {
+    name                           = "linux"
+    node_count                     = 1
+    vm_size                        = "Standard_DS2_v2"
+    os_disk_size_gb                = "50"
+    type                           = "VirtualMachineScaleSets"
+    enable_auto_scaling            = false
+    min_count                      = null
+    max_count                      = null
+    }
+}
+
+
+variable "additional_cluster_node_pools" {
+  
+  description = "The map object to configure one or several additional node pools with number of worker nodes, worker node VM size and Availability Zones."
+  type = object({
+    name                           = string
+    node_count                     = number
+    vm_size                        = string
+    os_type                        = string
+    os_disk_size_gb                = string
+    type                           = string
+    enable_auto_scaling            = bool
+    min_count                      = number
+    max_count                      = number
+  })
+    default = {
+    name                           = "windows"
+    node_count                     = 0
+    vm_size                        = "Standard_DS2_v2"
+    os_type                        = "Windows"
+    os_disk_size_gb                = "50"
+    type                           = "VirtualMachineScaleSets"
+    enable_auto_scaling            = false
+    min_count                      = null
+    max_count                      = null
+    }
+}
```

Comment:
Just a suggestion maybe it will make sense to use this:
https://www.terraform.io/docs/configuration/variables.html#variable-definitions-tfvars-files

Reply:
Those are good point. That being said, my opinion is that we should add another followup tasks for those 2 things @pprach mentioned and we should close this task and move to autoscaling.

Reply:
Yes, that's true! I was thinking of it but I wasn't sure if it is necessary. It is a easy and fast change which I've just done.. Could You take a look @mkyc and @pprach at last commit and approve and close the PR if everything is correct? 

Regarding to creating own module, I will discuss the topic with @toszo and create new ticket for it.